pyvista/core/pointset.py

```diff
@@ -1054,3 +1092,43 @@ def _get_attrs(self):
         attrs = PointGrid._get_attrs(self)
         attrs.append(("Dimensions", self.dimensions, "{:d}, {:d}, {:d}"))
         return attrs
+
+    def hide_cells(self, ind):
+        """Hide cells without deleting them.
+
+        Hides cells by setting the ghost_cells array to HIDDEN_CELL.
+
+        Parameters
+        ----------
+        ind : iterable
+            List or array of cell indices to be hidden.  The array can
+            also be a boolean array of the same size as the number of
+            cells.
+
+        Examples
+        --------
+        Hide part of the middle of a structured surface.
+
+        >>> import pyvista as pv
+        >>> import numpy as np
+        >>> x = np.arange(-10, 10, 0.25)
+        >>> y = np.arange(-10, 10, 0.25)
+        >>> z = 0
+        >>> x, y, z = np.meshgrid(x, y, z)
+        >>> grid = pv.StructuredGrid(x, y, z)
+        >>> grid.hide_cells(range(79*30, 79*50))
+        """
+        if isinstance(ind, np.ndarray) and ind.dtype == np.bool:
+            if ind.size != self.n_cells:
```

Comment:
This could now be a single if condition.

Reply:
See comment above.